### PR TITLE
Finished phonebook requirements and challenges

### PR DESCRIPTION
**Pull request by ukpagrace**

### CODE REVIEW

File: Phonebook.ukpagrace/Phonebook.ukpagrace/Mail.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+            string? password = config.GetSection("Email")["Password"];
+            MailMessage message = new MailMessage(
+                "noreply@gmail.com",
+                receiver,
+                "Phonebook",
+                $"You have been added to {receiver} phonebook"
+            );
+            SmtpClient smtpClient = new ("smtp.gmail.com");
+            NetworkCredential networkCredential = new NetworkCredential("ukpauchechi1@gmail.com", $"{password}");
```
Comment (javedkhan2k2) [inline Phonebook.ukpagrace/Phonebook.ukpagrace/Mail.cs:24]: You hardcoded your email here. Kindly move it to appsetting as well.

**Review by javedkhan2k2 — CHANGES_REQUESTED** [review]
@ukpagrace the basic functionality is working fine.
I will only suggest to improve the UI. Currently there is no Exit option in the Menu( You implemented it but user need to first select an option)
It will also be nice to select a User and send the email.
Kindly fix the Codacy issues (ignore the one with readme).

File: Phonebook.ukpagrace/Phonebook.ukpagrace/Mail.cs (hunk 1 above)
Comment (ukpagrace) [reply Phonebook.ukpagrace/Phonebook.ukpagrace/Mail.cs]: done,changed that

**Comment by ukpagrace** [comment]
> @ukpagrace the basic functionality is working fine. I will only suggest to improve the UI. Currently there is no Exit option in the Menu( You implemented it but user need to first select an option) It will also be nice to select a User and send the email. Kindly fix the Codacy issues (ignore the one with readme).


﻿using System.Net;
﻿if i remove this, it will throw an error

**Comment by javedkhan2k2** [comment]
@ukpagrace approved.
Best of luck for your next Project.

**Comment by ukpagrace** [comment]
> @ukpagrace approved.
> 
> Best of luck for your next Project.

Thank you

**Pull request closed without merging**